content/docs/5.1/develop/rawkv/cas.md

```diff
@@ -70,9 +70,9 @@ session.close();
 ```
 
 {{< warning >}}
-Users must set `conf.setEnableAtomicForCAS(true)` to ensure linearizability of `CAS` when used together with `put`, `delete`, `batch_put`, or `batch_delete`.
+You must set `conf.setEnableAtomicForCAS(true)` to ensure linearizability of `CAS` when using with `put`, `delete`, `batch_put`, or `batch_delete`.
 
 To guarantee the atomicity of CAS, write operations like `put` or `delete` in atomic mode are more expensive.
 {{< /warning >}}
 
-The code example used in this chapter can be found [here](https://github.com/marsishandsome/tikv-client-examples/blob/main/java-example/src/main/java/example/rawkv/CAS.java).
\ No newline at end of file
+The code example used in this chapter can be found [here](https://github.com/marsishandsome/tikv-client-examples/blob/main/java-example/src/main/java/example/rawkv/CAS.java).
```

Comment:
It is not recommended to link to a personal repo.

Reply:
ok, I will move this code to `https://github.com/tikv/website`.